mobile/src/actions/index.js

```diff
@@ -1 +1,63 @@
+import cozy, { LocalStorage as Storage } from 'cozy-client-js'
```

Comment:
This `LocalStorage` is used by *cozy-client-js* to store `clientID` and `clientSecret`.